pkg/cmd/spec/generator.go

```diff
@@ -3,7 +3,9 @@ package spec
 import (
 	"fmt"
 
+	"github.com/acarl005/stripansi"
 	"github.com/pterm/pterm"
+	"sigs.k8s.io/kustomize/kyaml/errors"
```

Comment:
why do we use a kustomize error instead of the native go errors

Reply:
I've changed to the native go errors package.